.github/workflows/integration-test.yaml

```diff
@@ -0,0 +1,72 @@
+name: Run integration tests
+
+on:
+  workflow_call:
+    inputs:
+      github_environment:
```

Comment:
Other than to echo out below is this used ? Typically we would use it to set env (eg to get secrets) ?

Reply:
No it is not used and neither is git_reference. Will remove both